controllers/admin/AdminTranslationsController.php

```diff
@@ -3308,4 +3310,29 @@ public function displayAjaxEmailHTML()
             AdminTranslationsController::getEmailHTML($email)
         );
     }
+
+    /**
+     * @param $directory : name of directory
+     *
+     * @return array
+     */
+    protected function getSubjectMailContent($directory)
+    {
+        $subject_mail_content = [];
+        if (Tools::file_exists_cache($directory . '/lang.php')) {
+            // we need to include this even if already included (no include once)
+            include $directory . '/lang.php';
+            foreach ($GLOBALS[$this->translations_informations[$this->type_selected]['var']] as $key => $subject) {
+                ++$this->total_expression;
+                $subject = str_replace('\n', ' ', $subject);
+                $subject = str_replace("\\'", "\'", $subject);
+                $subject_mail_content[$key]['trad'] = htmlentities($subject, ENT_QUOTES, 'UTF-8');
+                $subject_mail_content[$key]['use_sprintf'] = $this->checkIfKeyUseSprintf($key);
+            }
+        } else {
+            $this->errors[] = $this->trans('Email subject translation file not found in %path%', ['%path%' => $directory], 'Admin.International.Notification');
+        }
```

Comment:
@Amit-Kumar-Tiwari-Webkul Thank you for updating it with the "trans" method :)
However I couldn't find this string and its domain....
The closest one I found is: "_Email subject translation file not found in "%s"._"  and it is a string from the 1.6 version in Admin. Could you check the string and the domain again please?